web/src/composebox_typeahead.js

```diff
@@ -820,6 +821,21 @@ export function content_typeahead_selected(item, event) {
                     is_silent,
                 );
                 beginning += mention_text + " ";
+                const compose_stream_id = stream_data.get_stream_id(compose_state.stream_name());
+                const edit_stream_id = $textbox
+                    .closest(".recipient_row")
+                    .find(".message_header_stream")
+                    .attr("data-stream-id");
```

Comment:
What happens if you're editing a private message?

Reply:
Yeah, there will be an error since we don't have any information about the stream.

Currently, we have removed this logic because we don't want to check for wildcard mentions before we click "save" or "edit".